### PR TITLE
[@types/kalmanjs]: Add new package

### PR DESCRIPTION
**Pull request by pagyew**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test kalmanjs`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Adding a new definition:

- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/kalmanjs/kalmanjs-tests.ts
1. types/kalmanjs/index.d.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 6f50b13ad21508cfebbd3ebbe952b5aabea397bb
</p>

File: types/kalmanjs/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    setMeasurementNoise(noise: number): void;
+
+    /**
+     * Set the process noise R
+     * @param {Number} noise
+     */
+    setProcessNoise(noise: number): void;
+}
+
+export = KalmanFilter;
```
Comment (sheetalkamat) [inline types/kalmanjs/index.d.ts:82]: This shows usage in browser: https://www.npmjs.com/package/kalmanjs#in-the-browser So PR needs  update to also use as global: Refer to https://github.com/DefinitelyTyped/DefinitelyTyped?tab=readme-ov-file#how-do-i-write-definitions-for-packages-that-can-be-used-globally-and-as-a-module for making the change Add global test as well along with that

**Review by sheetalkamat — CHANGES_REQUESTED** [review]

File: types/kalmanjs/index.d.ts (hunk 1 above)
Comment (pagyew) [reply types/kalmanjs/index.d.ts:82]: I need some help. I've already looked at a bunch of other packages, and done everything similar, but CI keeps crashing. What am I doing wrong?

Comment (pagyew) [reply types/kalmanjs/index.d.ts:85]: It seems I finally was able to beat CI tests 🥵

**Review by RyanCavanaugh — APPROVED** [review]

**Comment by pagyew** [comment]
Ready to merge

**Pull request merged**